magefile.go

```diff
@@ -88,7 +88,7 @@ func (Test) Unit() error {
 }
 
 func (Test) Integration() error {
```

Comment:
I was not able to run this out of the box
```
Running target: Test:Integration
exec: go "test" "-v" "-failfast" "-count=1" "-timeout=20m" "./integration/..."
panic: creating runtime client: Get "<hive url>": dial tcp <ip-address>:443: i/o timeout

goroutine 1 [running]:
package-operator.run/package-operator/integration.init.0()
        /home/gathomas/GolandProjects/package-operator/package-operator/integration/environment.go:44 +0x168
FAIL    package-operator.run/package-operator/integration       30.067s
FAIL

```

Reply:
`./mage test:integration` needs a cluster already having PKO installed.
Try `./mage dev:integration` this will setup a new cluster with kind, compile and install PKO and then run the integration tests against it.